sklearn/linear_model/_ransac.py

```diff
@@ -306,7 +315,8 @@ def __init__(
         # RansacRegressor.estimator is not validated yet
         prefer_skip_nested_validation=False
     )
-    def fit(self, X, y, sample_weight=None):
+    @_deprecate_positional_args(version="1.7")
```

Comment:
Same comment as the other PR here, to make sure `sample_weight` is excluded from `fit_params` when this deprecation is over.